jetbrains-core/resources/telemetryOverride.json

```diff
@@ -219,12 +219,12 @@
             ]
         },
         {
-            "name": "codeTransform_ConfigurationFilePath",
+            "name": "codeTransformConfigurationFilePath",
```

Comment:
let's bring these into https://github.com/aws/aws-toolkit-common

Reply:
We talked about this pre launch. Can we do this as a follow up PR? Since release cycles are weekly and we are pushed to deliver this today instead of tomorrow.

Reply:
This will take time to test and deploy. If we can release tomorrow thats fine. But we were told it has to be done by today.

Reply:
Brought into commons package. DONE!